src/status_im2/contexts/quo_preview/settings/accounts.cljs

```diff
@@ -0,0 +1,48 @@
+(ns status-im2.contexts.quo-preview.settings.accounts
+  (:require [clojure.string :as string]
+            [quo2.components.settings.accounts.view :as quo2]
```

Comment:
You can simply alias quo2.core as `quo` so that the code is already future proof when we eliminate the old quo component library.

Reply:
Ok ok, I changed it in 46b87118c8aad9bf9f8fcc7c6fb9acf546cf59eb